schemas/Templatecontent.json

```diff
@@ -0,0 +1,76 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
+    "properties": {
+      "Templatecontent": {
+        "$ref": "#/definitions/Templatecontent"
+      }
+    },
+    "required": [
+      "Templatecontent"
+    ],
+    "title": "Templatecontent",
+    "definitions": {
+      "Templatecontent": {
+        "$id": "#/properties/Templatecontent",
+        "type": "object",
+        "title": "The Template content schema",
+        "description": "All Template content.",
+        "required": [],
+        "properties": {
+          "template_id": {
```

Comment:
Use camel casing for all. Also, this should be templateContentId right ? And additionally templateId needed as a reference from the other schema